ios/ZulipNative.xcodeproj/xcshareddata/xcschemes/ZulipNative.xcscheme

```diff
@@ -66,9 +66,9 @@
       </AdditionalOptions>
    </TestAction>
    <LaunchAction
-      buildConfiguration = "Debug"
-      selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
-      selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      buildConfiguration = "Release"
```

Comment:
Let's also remove this and keep the default config "Debug" (because using the simulator is more difficult in release mode)